src/main/resources/templates/CityDetails.html

```diff
@@ -107,6 +107,36 @@ <h1 id = "title">City</h1>
             </div>
             <br/><br/>
         </div>
+
+        <div class="">
+            <h3 th:text="'For a fabulous weekend getaway for 2 to ' + ${cityInfo.getName()} + ' signup here'"></h3>
+            <br/>
+            <div class="promotion">
+                <form th:action="@{/cities/reservation}" method="post">
+                    <div>
+                        <p> Email <input type="email" th:name="email"/></p>
+                        <p>Package
+                            <!--                            <ul>-->
+                        <div th:each="radioValue: ${levels}">
```

Comment:
I realized that you provided variable for levels in java code and use a loop to list them. Good job.